crash_similarity.py

```diff
@@ -15,6 +15,9 @@
 import download_data
 import utils
 
+from pyemd import emd
```

Comment:
Nit: Let's move this in the second block of imports, sorted in alphabetical order. The first block of imports is for built-in modules, the second block is for third-party modules, the third is for modules written by us.